go/conversation/tasks.py

```diff
@@ -90,19 +89,26 @@ def load_messages_in_chunks(conversation, direction='inbound',
         modified on the fly.
     """
     if direction == 'inbound':
-        bunches = conversation.mdb.inbound_messages.load_all_bunches(
-            conversation.inbound_keys())
+        index_page = conversation.mdb.batch_inbound_keys_page(
+            conversation.batch.key)
+        get_msg = conversation.mdb.get_inbound_message
     elif direction == 'outbound':
-        bunches = conversation.mdb.outbound_messages.load_all_bunches(
-            conversation.outbound_keys())
+        index_page = conversation.mdb.batch_outbound_keys_page(
+            conversation.batch.key)
+        get_msg = conversation.mdb.get_outbound_message
     else:
         raise ValueError('Invalid value (%s) received for `direction`. '
                          'Only `inbound` and `outbound` are allowed.' %
                          (direction,))
 
-    for messages in bunches:
+    while index_page is not None:
```

Comment:
Can we not do `while index_page.has_next_page():` and remove the if statement below?